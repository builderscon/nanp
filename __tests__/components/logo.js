/* eslint-env jest */

import React from 'react'
import Logo from '../../src/components/logo'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Logo />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
