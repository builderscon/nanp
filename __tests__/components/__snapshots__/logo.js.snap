exports[`test renders correctly 1`] = `
<Image
  source={1} />
`;
